docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -196,6 +196,298 @@ tcld apikey disable --id <api-key-id>
 tcld apikey enable --id <api-key-id>
 ```
 
+### API Keys for Client Connections
+Temporal Cloud supports secure programmatic access through API Key Authentication.
+
+
+API Keys for Client Connections are a new capability that extend the existing API Keys feature to afford users the ability to auth(n/z) into Temporal Cloud's "data plane" from Client Connections (Temporal CLI, SDKs) using an API Key.
+
+> Note: This feature capability changes the UX for Namespace creation/management. With API Keys for Client Connections, the connection method for a Namespace is configurable, either mTLS or API Key.
+
+#### Key Feature Terms
+
+- **API Key**: Authentication token that can be used in the Temporal CLI, SDKs, tcld, the Cloud Operations API, and the Terraform Provider.
+- **gRPC Endpoint**: The gRPC endpoint used in the Temporal CLI and/or SDKs to connect to Temporal Cloud.
+  - For an mTLS connection, the endpoint is in the format `<namespace>.<account>.tmprl.cloud:7233`
+  - For an API Key connection, the endpoint is in the format `<region>.<cloud provider>.api.temporal.io:7233`
+- **gRPC header**: Connection information supplied by the user in the Temporal CLI or SDK when connecting to Temporal Cloud using an API Key. Currently, in most SDKs and the CLI, the gPRC header should include a Bearer Token with the API Key and a Namespace header.
+
+## API Keys for Client Connections
+
+Temporal Cloud supports secure programmatic access through API Key Authentication.
+
+#### Overview
+
+Temporal Cloud API Keys facilitate identity-level authentication, for Users and Service Accounts. Each key is unique to a single identity and ties directly to the identity's Role-Based Access Control (RBAC) permissions, ensuring secure and appropriate access.
+
+The process is as follows:
+
+API Key (authentication) → User or Service Account (identity) → RBAC (authorization)
+
+These API Keys are used to authenticate Temporal Cloud operator tools, including Temporal Cloud CLI (tcld) and the Temporal Cloud Operations API (Cloud Ops API), and Client Connection tools, including the Temporal CLI and SDKs.
+
+
+#### API Key Summary
+
+Temporal Cloud API Keys functionality is designed for 3 types of use:
+
+1. User created API Keys - if enabled, users can create, delete, and update their own API Keys through the Cloud UI or tcld.
+2. Service Account created API Keys - Global Administrators can create, delete, and update API Keys for Service Accounts. 
+3. API Key governance - Temporal Cloud provides affordances for Global Administrators to govern and control API Key use within their accounts. Global Administrators can enable/disable API Keys for their accounts, and disable/delete API Keys for any User and/or Service Account.
+
+By default, API Keys must be enabled by a Global Administrator before use.
```

Comment:
might be worthwhile to highlight that this action is a "global" or account-level setting -- otherwise, users may have the impression that a global admin has to enable their personal API key before using it.